app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/controllers/TermsOfUseFailedJourneyController.scala

```diff
@@ -170,7 +200,89 @@ class TermsOfUseFailedJourneyController @Inject() (
     } yield ok
   }
 
+  def failOverridePage(applicationId: ApplicationId) = loggedInThruStrideWithApplicationAndSubmission(applicationId) { implicit request =>
+    successful(Ok(termsOfUseFailOverridePage(ViewModel(applicationId, request.application.name))))
+  }
+
+  def failOverrideAction(applicationId: ApplicationId) = loggedInThruStrideWithApplicationAndSubmission(applicationId) { implicit request =>
+    request.body.asFormUrlEncoded.getOrElse(Map.empty).get("override").flatMap(_.headOption) match {
+      case Some("yes") => successful(Redirect(uk.gov.hmrc.apigatekeeperapprovalsfrontend.controllers.routes.TermsOfUseFailedJourneyController.overrideApproverPage(applicationId)))
```

Comment:
Could we `import uk.gov.hmrc.apigatekeeperapprovalsfrontend.controllers.routes._` to keep this and similar lines shorter?

Reply:
Fixed